modules/update-upgrading-cli.adoc

```diff
@@ -13,54 +13,62 @@ You can find information about available {product-title} advisories and updates
 link:https://access.redhat.com/downloads/content/290[in the errata section]
 of the Customer Portal.
 
+[NOTE]
+====
+You can use the `--version` flag to determine whether a specific version is recommended for your update.
+====
+
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`) that matches the version for your updated version.
 * Log in to the cluster as user with `cluster-admin` privileges.
-
 * Pause all `MachineHealthCheck` resources.
-
+// Example output Failing=true taken from https://github.com/openshift/oc/blob/main/pkg/cli/admin/upgrade/recommend/examples/4.16.27-degraded-monitoring.output
```

Comment:
Something wrong here?
<img width="781" height="309" alt="image" src="https://github.com/user-attachments/assets/e3baae85-e21c-4968-8679-bb967cdb8e8a" />

Reply:
`Procedure` should be in a new line?